tm2/pkg/sdk/auth/params.go

```diff
@@ -20,11 +22,12 @@ const (
 
 // Params defines the parameters for the auth module.
 type Params struct {
-	MaxMemoBytes           int64 `json:"max_memo_bytes" yaml:"max_memo_bytes"`
-	TxSigLimit             int64 `json:"tx_sig_limit" yaml:"tx_sig_limit"`
-	TxSizeCostPerByte      int64 `json:"tx_size_cost_per_byte" yaml:"tx_size_cost_per_byte"`
-	SigVerifyCostED25519   int64 `json:"sig_verify_cost_ed25519" yaml:"sig_verify_cost_ed25519"`
-	SigVerifyCostSecp256k1 int64 `json:"sig_verify_cost_secp256k1" yaml:"sig_verify_cost_secp256k1"`
+	MaxMemoBytes           int64            `json:"max_memo_bytes" yaml:"max_memo_bytes"`
+	TxSigLimit             int64            `json:"tx_sig_limit" yaml:"tx_sig_limit"`
+	TxSizeCostPerByte      int64            `json:"tx_size_cost_per_byte" yaml:"tx_size_cost_per_byte"`
+	SigVerifyCostED25519   int64            `json:"sig_verify_cost_ed25519" yaml:"sig_verify_cost_ed25519"`
+	SigVerifyCostSecp256k1 int64            `json:"sig_verify_cost_secp256k1" yaml:"sig_verify_cost_secp256k1"`
+	UnrestrictedAddrs      []crypto.Address `json:"unrestricted_addrs" yaml:"unrestricted_addrs"`
```

Comment:
Just a note we don't initialize this new field in `DefaultParams`

Reply:
By default, there are no unrestricted accounts since token transfer locking is off. Unrestricted accounts should be added to the genesis file when token locking is set to true. These accounts can be added manually or through a separate genesis command later.